config/application.rb

```diff
@@ -1,6 +1,8 @@
 require File.expand_path('../boot', __FILE__)
 require 'rails/all'
 
+require 'raise-if-root'
```

Comment:
this will be included as part of the `Bundler.require` below right?

Reply:
Yeah I didn't really know what I was doing here. Bundler.require will require all the things?

Reply:
Yup! It requires all the things per environment (for example if you had included this gem in the "test" group of the Gemfile, it would only require it being installed/built for test, etc etc)